packages/graph/src/graph2/dataGraph.ts

```diff
@@ -43,8 +43,6 @@ export class DataGraph extends Graph2 {
     @publish("", "string", "Subgraph Label column")
     subgraphLabelColumn: publish<this, string>;
 
-    @publish([], "any", "Vertex Columns", null, { internal: true })
-    vertexColumns: publish<this, string[]>;
```

Comment:
Moved this to Graph2

Reply:
It doesn't belong in Graph2 as there is no concept of columns / data for the vertices,

IOW you should add the tooltip info to either the react Vertex or the IVertex placeholder (as an optional property).